package/visualization/plot_step_distribution/README.md

```diff
@@ -0,0 +1,50 @@
+---
+author: Hideaki Imamura
+title: Step Distribution Plot
+description: A visualization for the distribution of steps in the optimization if you use the pruning feature.
+tags: [visualization, pruning]
+optuna_versions: [3.6.1]
+license: MIT License
+---
+
+<!--
+This is an example of the frontmatters.
+All columns must be string.
+You can omit quotes when value types are not ambiguous.
+For tags, a package placed in
+- package/samplers/ must include the tag "sampler"
+- package/visualilzation/ must include the tag "visualization"
+- package/pruners/ must include the tag "pruner"
+respectively.
+
+---
+author: Optuna team
+title: My Sampler
+description: A description for My Sampler.
+tags: [sampler, 2nd tag for My Sampler, 3rd tag for My Sampler]
+optuna_versions: [3.6.1]
+license: "MIT License"
+---
+-->
+
+## Class or Function Names
+
+- `plot_step_distribution`
+
+## Installation
+
+You should install `plotly` to use this visualization.
+
+```bash
+$ pip install plotly
+```
+
+## Example
+
```

Comment:
I think it would be helpful to provide some explanations about this plot.
For example, "This plot shows how much pruning has been done so far."